### PR TITLE
docs: Slurm/PBS override of default resource pools

### PR DESCRIPTION
**Pull request by jerryharrow**
## Description

Add user doc for default_aux_resource_pool and default_compute_resource_pool.

image
and
image

Using the same phrasing from the Agent version of this setting:

image

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst [hunk 1]
```diff
@@ -125,7 +127,9 @@ to optimize how Determined interacts with PBS:
 
    -  A PBS queue with no GPUs is identified as an AUX resource pool.
 
-   -  The Determined default resource pool is set to the PBS default queue.
+   -  The Determined default resource pool is set to the PBS default queue. Override this default
+      using the :ref:`pbs section <cluster-configuration-slurm>` ``default_compute_resource_pool``
```
Comment (phillip-gaisford) [inline docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst:131]: Should that read '<cluster-configuration-pbs>'?

Comment (jerryharrow) [reply docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst:131]: I think your comment got truncated?   Not sure what you are asking.

Comment (phillip-gaisford) [reply docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst:131]: Yeah, something amiss there. 

You have cluster-configuration-slurm, but shouldn't that be cluster-configuration-_pbs_

Comment (jerryharrow) [reply docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst:131]: Nope.   This the resource_manager type is a single section in the master.yaml docs

.. _cluster-configuration-slurm:

   -  ``type: slurm`` or ``pbs``: The HPC launcher submits tasks to a Slurm/PBS cluster. For more
      information, see :ref:`using_slurm`.

**Review by phillip-gaisford — APPROVED** [review]

**Pull request merged**